tests/utils/nitpick.js

```diff
@@ -490,6 +511,7 @@ module.exports.addStepSnapshot = function (name, stepFunction) {
 // The default time between test steps may be modified through these methods
 module.exports.enableAuto = function () {
     testMode = "auto";
+    Window.stillSnapshotTaken.connect(onStillSnapshotTaken);

```

Comment:
This still seems wrong...why isn't it next to where you connect the key press event in `onRunAuto`?

Reply:
onRunAuto isn't called when running a recursive script.

Reply:
ah ok